meta/main.yml

```diff
@@ -26,9 +26,9 @@ dependencies:
   - role: stackhpc.os_openstacksdk
     os_openstacksdk_venv: "{{ os_images_venv }}"
     os_openstacksdk_state: "{{ os_images_package_state }}"
-    os_openstacksdk_upper_constraints_file: "{{ os_images_upper_constraints_file }}"
+    os_openstacksdk_version: "1.0"
 
   - role: stackhpc.os-openstackclient
     os_openstackclient_venv: "{{ os_images_venv }}"
     os_openstackclient_state: "{{ os_images_package_state }}"
-    os_openstackclient_upper_constraints_file: "{{ os_images_upper_constraints_file }}"
+    os_openstackclient_version: "6.2.0"
```

Comment:
I'm not sure this is the right way to go. Upper constraints help to keep installations repeatable, and this will ignore them.

Reply:
Replying to https://github.com/stackhpc/ansible-role-os-images/pull/77#discussion_r1297364996.

Looks like the Antelope (2023.1) upper constraints include openstacksdk 1.0.1 and openstackclient 6.2.0 https://opendev.org/openstack/requirements/raw/branch/stable/2023.1/upper-constraints.txt

I suggest we change the default of os_images_upper_constraints_file to use 2023.1

Reply:
@markgoddard done